src/unix/index_unix.ml

```diff
@@ -258,6 +258,7 @@ module IO : Index.IO = struct
 
   let clear ?(keep_generation = false) t =
     t.offset <- 0L;
+    t.fan_size <- 0L;
```

Comment:
Is this an unrelated fix?

Reply:
It's actually included in [`Fan.set`](https://github.com/mirage/index/blob/master/src/unix/index_unix.ml#L132), removed it as well.